src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
-body {
+@import '../styles/utils/variables';
+@import '../styles/blocks/card';
+@import '../styles/blocks/stars';
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
 }
```

Comment:
```suggestion

```
It's better to move general styles from the index file to a separate stylesheet to keep your index clean and clear.